flatpak/io.github.Pithos.json

```diff
@@ -1,7 +1,7 @@
 {
   "app-id": "io.github.Pithos",
   "runtime": "org.gnome.Platform",
-  "runtime-version": "master",
+  "runtime-version": "3.38",
```

Comment:
Why 3.38?

Reply:
Why not? the GNOME docker image could be used though which should work fine I guess. But as Pithos is not really targeting the latest GNOME, it makes sense to use the latest stable version.

Reply:
I agree we don't really target master, this seems reasonable.